app/main/views/services.py

```diff
@@ -14,7 +14,8 @@
     url_for, pagination_links, display_list, get_valid_page_or_1,
     validate_and_return_updater_request, get_int_or_400
 )
-
+from datetime import datetime
+from pendulum import Date
```

Comment:
Most of the examples in the Pendulum docs seem to prefer `import pendulum` and using `pendulum` directly for `.today()`, `.tomorrow()` etc.

Also, `isinstance(pendulum.utcnow(), datetime) === True` so you probably don't need `datetime` at all - e.g. `pendulum.utcnow()` could replace the `datetime.utcnow()` uses.